strange_function.py

```diff
@@ -0,0 +1,34 @@
+# HAcker Rank Problem: https://www.hackerrank.com/challenges/the-strange-function/problem
+#!/bin/python3

```

Comment:
This should be the first line in the file. The magic hashbang/shebang has to be the first byte in the file for it to work:

https://stackoverflow.com/questions/12910744/why-should-the-shebang-line-always-be-the-first-line